atc/db/resource_cache.go

```diff
@@ -61,20 +61,22 @@ func (cache *ResourceCacheDescriptor) findOrCreate(
 			Columns(
 				"resource_config_id",
 				"version",
```

Comment:
Shouldn't we remove this reference to `version`?

Reply:
After discussion we found it would be easier to keep both version and version_md5 in the table. 
For example, in https://github.com/concourse/concourse/blob/9613e27cf55d58b81dd5439d0f05c93d721c5204/atc/db/resource_cache_factory.go#L128

if `version` column is dropped then it will need to make a query to resource_config_versions table with `version_md5`, it is not clear that if the version in  `resource_config_versions` is always sync with `resource_caches` table  in all cases.